sunpy/map/sources/suvi.py

```diff
@@ -1,19 +1,18 @@
 """SUVI Map subclass definitions"""
-from __future__ import absolute_import, print_function, division
+from __future__ import absolute_import, division, print_function
```

Comment:
what the dammned hell is this still doing here?!

Reply:
Removed in my local version.